src/openrct2/ride/RideConstruction.h

```diff
@@ -62,7 +62,7 @@ extern TrackPitch _previousTrackPitchEnd;
 
 extern CoordsXYZ _previousTrackPiece;
 
-extern uint8_t _currentBrakeSpeed2;
+extern int8_t _currentBrakeSpeed;
```

Comment:
Why change this to a signed integer?

Reply:
It was a change related to https://github.com/OpenRCT2/OpenRCT2/pull/16692#issuecomment-1801094250 that leaked into the original feature branch.

Reply:
Ok, can you revert it? I want to keep the diff as small as possible.

Reply:
I can, yes, but not for the next several days since I am very busy. I am still waiting on a decision regarding the 2 bits in the flags field as outlined in that comment.

Reply:
Sorry, didn’t know you expected me to weigh in on it. I’ll get to it.